tests/test_operations.py

```diff
@@ -65,6 +61,30 @@ def test_delete(self, requests_mock):
         r = operations.delete(data_url)
         assert r is None
 
+    # test 'format' parameter
+    def test_create_format(self, data, requests_mock):
+        """Test create function parameter `format` overwriting `content-type`"""
+        requests_mock.post(base_url)
+        r = operations.create(base_url, data=data,
+            **{'headers': {'content-type': 'application/ld+json'}})
+        assert r is None
```

Comment:
Does this actually check that the header was overridden? Or does the `requests_mock` call on line 67 do that? I can't see how it would know the intended behaviour...

Reply:
Should add a mock of headers in line 67.